Homeworks/Homework2/src/tests/demoblaze_test.py

```diff
@@ -0,0 +1,31 @@
+from Homeworks.Homework2.src.main.config import user_name, user_password
+
+
+class TestDemoBlaze:
+    # Test: login
+    def test_login(self, demo_blaze_page):
+        demo_blaze_page.click_login_button()
+        demo_blaze_page.setup_login_and_password(user_name, user_password)
+        demo_blaze_page.verify_login(user_name)
+
+    # Test: add to cart
+    def test_add_to_cart(self, demo_blaze_page, item_page, cart_page):
+
+        # Step 1: Login
+        demo_blaze_page.click_login_button()
+        demo_blaze_page.setup_login_and_password(user_name, user_password)
```

Comment:
login part in this case is considered to be a part of precondition, not a test step (it would be better implement it as a fixture)

Reply:
thank you :)